File: sdks/sdkwork-control-plane-sdk/sdkwork-control-plane-sdk-typescript/composed/src/meta-module.ts
import type { JsonObject } from './types.js';
import type { ControlPlaneSdkContext } from './sdk-context.js';

export class ControlPlaneMetaModule {
  constructor(private readonly context: ControlPlaneSdkContext) {}

  health(): Promise<JsonObject> {
    return this.context.backendClient.meta.getHealthz();
  }
}

File: sdks/sdkwork-control-plane-sdk/sdkwork-control-plane-sdk-flutter/composed/lib/src/meta_module.dart
import 'package:control_plane_backend_sdk/control_plane_backend_sdk.dart';

import 'context.dart';

class ControlPlaneMetaModule {
  final ControlPlaneSdkContext context;

  ControlPlaneMetaModule(this.context);

  Future<JsonObject> health() {
    return context.backendClient.meta.getHealthz();
  }
}
